package.json

```diff
@@ -100,14 +100,14 @@
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.11.2",
     "@hms-dbmi-bgm/react-workflow-viz": "0.1.4",
-    "@hms-dbmi-bgm/shared-portal-components": "github:4dn-dcic/shared-portal-components#0.0.2.61.3",
+    "@hms-dbmi-bgm/shared-portal-components": "github:4dn-dcic/shared-portal-components#0.0.2.62.3b",
     "4dn-microscopy-metadata-tool": "github:WU-BIMAC/4DNMicroscopyMetadataToolReact#0.0.0.18",
     "babel-polyfill": "^6.26.0",
     "d3": "^5.12.0",
     "detect-browser": "^3.0.1",
     "domready": "^0.3.0",
     "form-serialize": "^0.6.0",
-    "higlass": "^1.6.11",
+    "higlass": "^1.9.3",
```

Comment:
Noticed this was already being updated to 1.9.x in package-lock.json so bumped it up here to match.